var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -25,25 +25,27 @@
 from spack import *
 
 
-class Libint(Package):
+class Libint(AutotoolsPackage):
     """Libint is a high-performance library for computing
-    Gaussian integrals in quantum mechanics."""
+    Gaussian integrals in quantum mechanics.
+    """
 
     homepage = "https://github.com/evaleev/libint"
-    url      = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
+    url = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
 
+    version('2.2.0', 'da37dab862fb0b97a7ed7d007695ef47')
     version('2.1.0', 'd0dcb985fe32ddebc78fe571ce37e2d6')
     version('1.1.6', '990f67b55f49ecc18f32c58da9240684')
     version('1.1.5', '379b7d0718ff398715d6898807adf628')
 
     # Build dependencies
     depends_on('autoconf@2.52:', type='build')
-    depends_on('automake',       type='build')
-    depends_on('libtool',        type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```

Comment:
PEP8 is silent on these spacing issues, and (AFAIK) Spack has not taken an official position on them.  As long as either way is OK, then changing it yields no objective improvement in the code (objective == Spack standards).  I believe we should avoid such edits in our PRs.  Not the least because they could result in "edit wars" in which certain lines keep getting changed back and forth, depending on the last person who edited them.  (For example, there are other PRs that end up ADDING spaces where this PR is REMOVING spaces).